bin/crew

```diff
@@ -490,7 +529,6 @@ def update
   puts 'Package lists, crew, and library updated.'
 
   #update compatible packages
-  puts 'Generating compatible packages...'
   generate_compatible
   puts 'Generating compatible packages done.'.orange if @opt_verbose
```

Comment:
Does it make sense to add this message to `generate_compatible` since it's called multiple times?

Reply:
`generate_compatible` is only called only once during crew operations, unless you are seeing something I'm missing.
once during `crew update`
once during `crew update compatible`
once during `crew list compatible`
once during `crew list incompatible`

Reply:
Right but my point is this message will only print for `crew update` and not `crew list ...`.

Reply:
Is that a problem? If you're asking crew to list compatible or incompatible packages, you really can't complain if it then does that...

Let me put it behind a verbose debug flag...

Reply:
I just thought it makes sense so the message is displayed whenever it's called.

Reply:
I misunderstood. It **was** being printed whenever it was being called, as I just moved that puts line to generate_compatible.
If you do `crew -v` something that invokes it, it will print that line...

Reply:
Right but it only prints that line with `crew update -v`; not with `crew list compatible -v`.

Reply:
I'm seeing different behavior:
```
chronos@localhost ~/src/satadru/chromebrew (crewrescue)$ crew -v list compatible
Generating compatible packages...
Checking a2png for compatibility.
```

Also
```
crew list compatible -v
Generating compatible packages...
Checking a2png for compatibility.
```

Reply:
Do you see `Generating compatible packages done.` with `crew -v list compatible`?

Reply:
Good catch! Fixed that by moving that inside `generate_compatible`.